From 6c24b45f8247667e3404b3110c72a062926cbd30 Mon Sep 17 00:00:00 2001
From: Lu Wei <luwei32@huawei.com>
Date: Tue, 15 Nov 2022 11:44:11 +0800
Subject: [PATCH 0073/1398] [Huawei] tcp_comp: modify default value of TCP_COMP
 to n

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187986

-------------------------------

Modify default value of TCP_COMP to n.

Signed-off-by: Lu Wei <luwei32@huawei.com>
---
 net/ipv4/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/ipv4/Kconfig b/net/ipv4/Kconfig
index 0ce3f61658b7..b44e3cf02589 100644
--- a/net/ipv4/Kconfig
+++ b/net/ipv4/Kconfig
@@ -747,6 +747,7 @@ config TCP_COMP
 	bool "TCP: Transport Layer Compression support"
 	depends on CRYPTO_ZSTD=y
 	select STREAM_PARSER
+	default n
 	help
           Enable kernel payload compression support for TCP protocol. This allows
           payload compression handling of the TCP protocol to be done in-kernel.
-- 
2.17.1

